tests/bootstrap.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
There is a bootstrap environment in `tests/php/bootstrap.php` that `phpunit.xml.dist` references. What is calling/using this file? If that needs modifying, let's do it there.